eo-maven-plugin/src/test/java/org/eolang/maven/FakeMaven.java

```diff
@@ -395,6 +397,16 @@ private FakeMaven withProgram(final String content) throws IOException {
         return res;
     }
 
+    /**
+     * Returns the current scope that was set.
+     * @return The current scope.
+     */
+    private String scope() {
+        return Optional.ofNullable(this.params.get("scope"))
```

Comment:
@volodya-lombrozo Can get rid of nullable value here? It would be more convenient to set explicitly default value of the scope.

Reply:
@mximp Done. Thank you.